### DISCUSSION TITLE
Regular Zoom Community Dev Catch-up

### DISCUSSION THREAD

**Discussion by ian-jorgensen** [SynthstromAudible/DelugeFirmware#85] [General]
Hey all!

I wanted to suggest the idea of a reasonably regular zoom chat to keep on top of things.
I think weekly on Sundays at 3pm EDT, could work. I reckon I could host this 2-3 times a month. Limit it to 30mins a time.
It's a time that is mid-evening in UK/EU and afternoon in US/CAN. it sucks for people in Asia/Australia, but there is obviously no single time which is going to suit everyone.
The chat won't be live streamed or recorded, its just so we can et together regularly and talk over key items. we'll try keep a kinda minutes of the chat so those who miss out, can keep up.

If this seems a plan we can do our first one on June 25th at 3pm. I'll share a Zoom invite link on this thread.

A plan?

**Comment by litui** [comment]
Great idea!

**Comment by jamiefaye** [comment]
I agree with Aria that this is a good idea. The time should work for me too.

– Jamie

**Comment by PaulFreund** [comment]
Love the idea!

**Comment by weavermedia** [comment]
Awesome! 🙌

**Comment by alex-r-t** [comment]
Would it make sense to alternate the time every other week? 3 PM EDT one week, something like 9 PM EDT another? I guess it depends on if there's folks that say they can't make it. I think it's a great idea otherwise! I'll be there either way 😁

**Comment by ian-jorgensen** [comment]
Zoom link: https://us06web.zoom.us/j/87166906136 (3pm EDT Sunday 25th June)

**Comment by litui** [comment]
I won't be able to make it today unfortunately. Family gathering ^_^


------- Original Message -------

**Comment by ian-jorgensen** [comment]
I am going back to bed for some more sleep. I'll get a link to the recording up here sometime in the next 24 hours.

Thanks everyone! cya next week :)

**Reply by alex-r-t** [reply]
@ian-jorgensen Any updates on the link?

**Comment by ian-jorgensen** [comment]
Link to chat number 1 here. This is unlisted. Please just keep these videos within the GitHub community. Would prefer these not distributed widely: https://youtu.be/a-3EIEJSyY8

**Comment by ian-jorgensen** [comment]
Meeting #2, same time, same place, same link. Gotta keep this one to a tight 30 or less as I'm on-the-road :)
https://us06web.zoom.us/j/87166906136 3pm EDT Sunday

**Comment by ian-jorgensen** [comment]
Sorry everyone, I'm in Australia at a music festival and can't host this week, but will be back same time same place next week :)

**Comment by ian-jorgensen** [comment]
I'm able to do the chat this week if keen. 
3:30pm EDT Sunday

**Comment by ian-jorgensen** [comment]
Chat link for sesh today; [https://us06web.zoom.us/j/87166906136](https://www.google.com/url?q=https://us06web.zoom.us/j/87166906136&sa=D&source=calendar&usd=2&usg=AOvVaw0bDx0Q_7pZb1Qe8XhuLE3B)

**Comment by jamiefaye** [comment]
Here is where to get a SysEx ID:
https://midimanufacturersassociation.formstack.com/forms/sysex_id_new_app

**Comment by ian-jorgensen** [comment]
As discussed. Tomorrow's meetup will be in the Discord Channel (trying that out). Somebody please pop a link here :)
Sunday 3pm EDT

**Reply by PaulFreund** [reply]
https://discord.com/channels/608916579421257728/608920177932763149 here's the link :)

**Reply by ian-jorgensen** [reply]
I won't be able to make this one :/ can't sleep...still up and this 5.5 hours away :( 

Will try jump in the dev chat when I do wake up properly...if I do sleep at some point
